src/actions/bannerActions.js

```diff
@@ -4,29 +4,31 @@ import '@firebase/storage';
 import { FETCH_BANNER } from './types';
 
 const config = {
-  apiKey: process.env.REACT_APP_API_KEY,
-  authDomain: process.env.REACT_APP_AUTH_DOMAIN,
-  databaseURL: process.env.REACT_APP_DATABASE_URL,
-  projectId: process.env.REACT_APP_PROJECT_ID,
-  storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
-  messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
+  apiKey: 'AIzaSyAgZpFs0epMtMtAPUE3CZ7M-jDPseMB5Dc',
+  authDomain: 'evernote-clone-80461.firebaseapp.com',
+  databaseURL: 'https://evernote-clone-80461.firebaseio.com',
+  projectId: 'evernote-clone-80461',
+  storageBucket: 'evernote-clone-80461.appspot.com',
+  messagingSenderId: '973863854205',
+  appId: '1:973863854205:web:0c40546f7e7d9e8a195114',
+  measurementId: 'G-70EYP43Q18',
 };
 
 firebase.initializeApp(config);
 
-const storage = firebase.storage();
+// const storage = firebase.storage();
 
 export const fetchBanner = () => dispatch => {
-  storage
-    .ref(`/banner.png`)
-    .getDownloadURL()
-    .then(url =>
-      dispatch({
-        type: FETCH_BANNER,
-        payload: url,
-      })
-    )
-    .catch(err => {
-      console.error('Failed to load banner image');
-    });
+  // storage
+  //   .ref(`/banner.png`)
+  //   .getDownloadURL()
+  //   .then(url =>
+  //     dispatch({
+  //       type: FETCH_BANNER,
+  //       payload: url,
+  //     })
+  //   )
+  //   .catch(err => {
+  //     console.error('Failed to load banner image');
+  //   });
```

Comment:
This is something I should have brought up at the meeting today so my bad. We need to work towards an admin page for uploading a new cover photo but could you un-comment out the code here

Reply:
Just did you can check it

Reply:
Also, I accidentally leaked my own keys so I had to amend and force-push